lib/mysqlDaoQueryHelper.js

```diff
@@ -25,6 +25,8 @@ class MysqlDaoQueryHelper {
             lte: '<=',
             eq: '=',
             ne: '!=',
+            in: 'IN',
+            nin: 'NOT IN',
```

Comment:
following up and extrapolating where this is likely going. I could see `between` and `like` operators getting added...following the `nin` pattern this would make them `nbetween` and `nlike` rather than (what i would prefer) `notBetween` and `notLike`.